test/e2e/projects_test.go

```diff
@@ -38,7 +38,7 @@ const (
 	nonExistingProjectName = "nonExistingProjectNameCAPX"
 )
 
-var _ = Describe("Nutanix projects", func() {
+var _ = Describe("Nutanix projects", Label("capx-feature-test", "slow", "network"), func() {
```

Comment:
Please add a specific label so we can target the specific project tests

Reply:
added